.travis.yml

```diff
@@ -7,7 +7,7 @@ services:
 # https://github.com/INCATools/ontology-starter-kit/issues/25
 # EMAIL_ARGS='-e obo-ci-reports-all@groups.io'
 script:
-  - make docker-test
+  - make tests
```

Comment:
travis.yml should be deleted altogether

Reply:
I was not sure about that since I have never used Travis at all. Is deleting that file enough to cut any link with Travis?

Reply:
I will cut ties with travis entirely now, you can delete the file.